docs/api.md

```diff
@@ -1661,6 +1668,13 @@ This method returns all of the dedicated [WebWorkers](https://developer.mozilla.
 
 > **NOTE** This does not contain ServiceWorkers
 
+### class: PageEvent
```

Comment:
Same here. What about NewPageEvent?

Reply:
I'd keep this one as is as it's shorter. Happy to rename if others prefer NewPageEvent or PageCreatedEvent.